### ISSUE TITLE
Add sorting to results.xml - by longest running tests etc.

### ISSUE THREAD

**Issue by OndraM** [lmc-eu/steward#133]
Results.xml file (when viewed in browser) applies [XSL template](https://github.com/lmc-eu/steward/blob/master/src/Resources/results.xsl) to format the HTML. 

![screenshot_20170505_135841](https://cloud.githubusercontent.com/assets/793041/25744583/f3d3fa60-319a-11e7-8c09-61882de1727d.png)

We already use moment.js library to format date and to calculate the diff between start and end of the tests. Another nice feature could be to sort the table by some of columns:
- status
- result
- start time
- end time
- duration

For example [jQuery Tablesorter 2](https://mottie.github.io/tablesorter/docs/) plugin could be used. It also contains some [Bootstrap 3](https://mottie.github.io/tablesorter/docs/example-option-theme-bootstrap-v3.html) theme.

**To summarize the requirements for this feature:**
- sort (only) by columns specified above
- show sort icon
- add ability to reset the filters (and return table to its original state)
- use bootstrap  theme (or similar - doen't have to be exactly the one) to keep it consistent with the rest of the results.xml
